### PR TITLE
Feature/gotomeeting

### PR DESCRIPTION
**Pull request by ericbieller**

### CODE REVIEW

File: comparison/speak-vs-gotomeeting/index.html [hunk 1]
```diff
@@ -0,0 +1,62 @@
+---
+window_title: How does Speak compare against GoToMeeting?
+title: Speak vs. Gotomeeting
+subtitle: Find out how Speak compares to GoToMeeting
+header_description: false
+header_image: speak-vs-gotomeeting
+header_pattern: pattern-1
+color: 2793B1`
```
Comment (tommoor) [inline comparison/speak-vs-gotomeeting/index.html]: hanging char here

File: comparison/speak-vs-gotomeeting/index.html [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+    <p>
+       If you're looking for quick, daily team 1-on-1 and group communication, GoToMeeting just doesn't cut it. Speak, on the other hand, was built from the ground up to make daily communication a breeze for large and small teams alike.
+     </p>
+     <p>
+       Our interface is designed to stay open all day, without getting in your way. This means that your team is always at your finger tips. Speak even knows if they've stepped away from their computer!
+    </p>
+    
+    <h2>Speak is lighter</h2>
+    <p>
+      GoToMeeting certainly has a lot of bells and whistles, but when you just want to have your meeting and get on with your day, these "features" just get in the way and make things confusing.
```
Comment (tommoor) [inline comparison/speak-vs-gotomeeting/index.html]: Lets discuss this para before merge, I think we can make it much more powerful - sounds like we're trying to explain away our lack of features a little :smile:

**Comment by ericbieller** [comment]
@tommoor just removed top paragraph, I think it works a lot better now

**Comment by tommoor** [comment]
Lets merge to staging, I feel like it's easier to review the copy on the staging site and make any tweaks when we meet later

**Pull request merged**